app/styles/templates/project/index.scss

```diff
@@ -5,10 +5,23 @@
 
 .project-main {
   @include span-columns(8);
+  @include media($lg-screen) {
```

Comment:
Just for my knowledge, where does the `media($lg-screen)` come into play? I see from the `_breakpoints.scss` its defining sizes but how does it recognize/render this?